classes/locations.php

```diff
@@ -238,18 +238,18 @@ function geo_location( $post ) {
 			$location_lng      = get_post_meta( $post->ID, 'location_lng', true ) ?: '';
 
 			?>
-			<p class="sub"><?php _e( 'You must enter either an address or a latitude/longitude. If you enter both, the address will not be geocoded and your latitude/longitude values will remain intact.', 'simplemap' ); ?></p>
+			<p class="sub"><?php _e( 'You must enter either an address or a latitude/longitude. If you enter both, the address will not be geocoded and your latitude/longitude values will remain intact.', 'SimpleMap' ); ?></p>
 
 			<div class='hidden updated below-h2' id='js-geo-encode-msg'>
-				<p><?php echo __( "Your Server's IP is over the geocode threshold set by Google so we had fallback to a Javascript function. <span id='sm_js_update_lat_lng_result'></span>", 'simplemap' ); ?></p>
+				<p><?php echo __( "Your Server's IP is over the geocode threshold set by Google so we had fallback to a Javascript function. <span id='sm_js_update_lat_lng_result'></span>", 'SimpleMap' ); ?></p>
 			</div>
 
 			<div class="table">
 				<table class="form-table">
 
 					<!-- Store Address -->
 					<tr valign="top">
-						<td width="150"><label for="location_address"><?php _e( 'Address', 'simplemap' ); ?></label></td>
+						<td width="150"><label for="location_address"><?php _e( 'Address', 'SimpleMap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 121 characters

<!-- Comment by [SideCI](https://sideci.com) -->